troubleshoot-lock-conflicts.md

```diff
@@ -230,4 +230,147 @@ When a pessimistic transaction has a deadlock, one of the transactions must be t
+                ID: 425496938634543111
+        START_TIME: 2021-06-08 08:46:48.341000
+CURRENT_SQL_DIGEST: a4e28cc182bdd18288e2a34180499b9404cd0ba07e3cc34b6b3be7b7c2de7fe9
+             STATE: LockWaiting
+WAITING_START_TIME: 2021-06-08 08:46:48.388024
+   MEM_BUFFER_KEYS: 1
+  MEM_BUFFER_BYTES: 19
+        SESSION_ID: 87
+              USER: root
+                DB: test
+   ALL_SQL_DIGESTS: [0fdc781f19da1c6078c9de7eadef8a307889c001e05f107847bee4cfc8f3cdf3, a4e28cc182bdd18288e2a34180499b9404cd0
+ba07e3cc34b6b3be7b7c2de7fe9, a4e28cc182bdd18288e2a34180499b9404cd0ba07e3cc34b6b3be7b7c2de7fe9]
+*************************** 2. row ***************************
+                ID: 425496940994101249
+        START_TIME: 2021-06-08 08:46:57.342000
+CURRENT_SQL_DIGEST: a4e28cc182bdd18288e2a34180499b9404cd0ba07e3cc34b6b3be7b7c2de7fe9
+             STATE: LockWaiting
+WAITING_START_TIME: 2021-06-08 08:46:57.590060
+   MEM_BUFFER_KEYS: 0
+  MEM_BUFFER_BYTES: 0
+        SESSION_ID: 85
+              USER: root
+                DB: test
+   ALL_SQL_DIGESTS: [0fdc781f19da1c6078c9de7eadef8a307889c001e05f107847bee4cfc8f3cdf3, a4e28cc182bdd18288e2a34180499b9404cd0
+ba07e3cc34b6b3be7b7c2de7fe9]
+2 rows in set (0.00 sec)
+```
+
+#### A transaction is blocked for a long time
+
+If a transaction is known to be blocked by another transaction (or multiple transactions) and the `start_ts` (transaction ID) of the current transaction is known, you can use the following method to obtain the information of the blocking transaction. Note that when you join multiple tables in Lock View, the data in different tables might not be obtained at the same time, so the information in different tables might not be consistent.
+
+{{< copyable "sql" >}}
+
+```sql
+select l.key, trx.* from information_schema.data_lock_waits as l join information_schema.tidb_trx as trx on l.current_holding_trx_id = trx.id where l.trx_id = 425497223886536705\G
+```
+
+```sql
+*************************** 1. row ***************************
+               key: 7480000000000000475f728000000000000002
+                ID: 425497219115778059
+        START_TIME: 2021-06-08 09:04:38.292000
+CURRENT_SQL_DIGEST: a4e28cc182bdd18288e2a34180499b9404cd0ba07e3cc34b6b3be7b7c2de7fe9
+             STATE: LockWaiting
+WAITING_START_TIME: 2021-06-08 09:04:38.336264
+   MEM_BUFFER_KEYS: 1
+  MEM_BUFFER_BYTES: 19
+        SESSION_ID: 97
+              USER: root
+                DB: test
+   ALL_SQL_DIGESTS: [0fdc781f19da1c6078c9de7eadef8a307889c001e05f107847bee4cfc8f3cdf3, a4e28cc182bdd18288e2a34180499b9404cd0
+ba07e3cc34b6b3be7b7c2de7fe9, a4e28cc182bdd18288e2a34180499b9404cd0ba07e3cc34b6b3be7b7c2de7fe9]
+1 row in set (0.01 sec)
+```
+
+If the `start_ts` of the current transaction is unknown, you can try to find out the information of the blocking transaction in the `TIDB_TRX` / `CLUSTER_TIDB_TRX` table or [`PROCESSLIST` / `CLUSTER_PROCESSLIST`](/information-schema/information-schema-processlist.md) table.
```

Comment:
```suggestion
If the `start_ts` of the current transaction is unknown, you can try to find it out from the information in the `TIDB_TRX` / `CLUSTER_TIDB_TRX` table or [`PROCESSLIST` / `CLUSTER_PROCESSLIST`](/information-schema/information-schema-processlist.md) table.
```

Actually I mean trying to find out the `start_ts`  of the current transaction, and the current transaction is the blocked transaction rather than blocking transaction.